test/e2e/support/hooks.js

```diff
@@ -115,9 +115,9 @@ defineSupportCode(({ Before,After }) => {
                 }
                 CucumberReportLog.AddJson(browserErrorLogs);
             } else {
-                await browser.manage().logs().get('browser');
+                browser.manage().logs().get('browser');
                 await CucumberReportLog.AddMessage("Cleared browser logs after successful scenario.");
-                await CucumberReportLog.AddScreenshot(global.screenShotUtils);
+                // await CucumberReportLog.AddScreenshot(global.screenShotUtils);
```

Comment:
Is this still required? Looks like this line was previously deleted but is now reinstated, albeit commented out.

Reply:
@sreekanthpuligadda?

Reply:
addressed. removed screenshot capture on scenario on success. cleaned commented code.